src/main/java/RacingCar.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCar {
+
+    public static void main(String[] args) {
+
+        int carCount = InputView.getAnswerToInteger("자동차 대수는 몇 대 인가요?");
+
+        List<Car> cars = generateCars(carCount);
+
+        int tryTimes = InputView.getAnswerToInteger("시도할 회수는 몇 회 인가요?");
+
+        ResultView.println("실행 결과");
+
+        for (int j = 0; j < tryTimes; j++) {
+            moveCars(cars);
+            ResultView.println("");
```

Comment:
```suggestion
            moveCars(cars);
```

`ResultView.printResult(car);` 에서 개행을 내부적으로 추가호출을 해도 좋을 것 같아요 🤔